data/experiment-recipe-samples/desktop-95-multifeature.json

```diff
@@ -0,0 +1,72 @@
+{
+  "appId": "firefox-desktop",
+  "appName": "firefox_desktop",
+  "application": "firefox-desktop",
+  "arguments": {},
+  "branches": [
```

Comment:
Do desktop experiments not use the feature key anymore? If they will not, can older single feature experiments be affected by this?

Reply:
Yep, should definitely have the `feature` key in there 👍